gateway/api/connections/connections.go

```diff
@@ -32,6 +33,18 @@ type Review struct {
 	ApprovalGroups []string `json:"groups"`
 }
 
+// Estruturas para respostas dos novos endpoints
+type TablesResponse struct {
+	Schemas []struct {
+		Name   string   `json:"name"`
+		Tables []string `json:"tables"`
+	} `json:"schemas"`
```

Comment:
Don't use inline struct, define each type accordingly. It's harder to read these definition throughout the code.